tools/c7n_mailer/c7n_mailer/cli.py

```diff
@@ -10,8 +10,10 @@
 from c7n_mailer import deploy, utils
 from c7n_mailer.azure.azure_queue_processor import MailerAzureQueueProcessor
 from c7n_mailer.azure import deploy as azure_deploy
+from c7n_mailer.gcp.gcp_pubsub_processor import MailerGcpPubSubProcessor
 from c7n_mailer.sqs_queue_processor import MailerSqsQueueProcessor
-from ruamel import yaml
+from ruamel import yml
```

Comment:
this import doesn't look right, looks like you did a find and replace on yaml/yml

Reply:
Yeah it was a typo on my part. I accidentally removed that import and typed it back in wrong. Fixed in next commit.